app/main/index.ts

```diff
@@ -262,6 +263,10 @@ app.on('ready', () => {
 		page.send('toggle-autohide-menubar', showMenubar, true);
 	});
 
+	ipcMain.on('toggle-sidebar', () => {
```

Comment:
Do we need to do similar thing for app window resize (or is it done already)?

Reply:
```
view.setAutoResize({ width: true, height: true });
```
This line in fixBounds() ensured automatic resizing with window.

Reply:
Great!